webmidi/midiinput_onmidimessage.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>EventHandler of onmessage() in MIDIInput interface | Web MIDI API Test</title>
+    <link rel="author" title="Ryoya KAWAI" href="mailto:ryoya.kawai@gmail.com">
+    <link rel="help" href="http://webaudio.github.io/web-midi-api/#midimessageevent-interface"> <!7.>
+    <link rel="match" href="reference/REFERENCE-FILE-NAME">
+    <meta name="assert" content="Return true when test success in inputs().">
+    <meta name="assert" content="Success when obtain over <span id='ulimit'></span> MIDI messages of noteOn or noteOff over onmidimessage().">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+
+    <style type="text/css">
```

Comment:
No, it does not necessary to display in Arial font.
Deleted.
https://github.com/ryoyakawai/web-platform-tests/commit/4b645b9dc7ecb418f0748dc737b26369c62d61b4

Thanks!